include/SDL3/SDL_blendmode.h

```diff
@@ -52,7 +52,8 @@ typedef enum
                                               dstA = dstA */
     SDL_BLENDMODE_MUL = 0x00000008,      /**< color multiply
                                               dstRGB = (srcRGB * dstRGB) + (dstRGB * (1-srcA))
-                                              dstA = (srcA * dstA) + (dstA * (1-srcA)) */
+                                              dstA = (srcA * dstA) + (dstA * (1-srcA))
```

Comment:
This should just say dstA = dstA, like the other operations

Reply:
yes